rmf_map_msgs/msg/Graph.msg

```diff
@@ -0,0 +1,4 @@
+string name
```

Comment:
I think the map server are no longer required to know about the navigation nodes so all the Graph* messages should be removed?

Reply:
It looks to me like the intention is for the `Graph.msg` to be used to express

1. The layout of the walls in the environment
2. The layout of each fleet's navigation graph

It's true that we no longer have a single metamap navigation graph, because we're allowing every fleet to have its own navigation graph, but it may still be useful to communicate all the various fleet navigation graphs to the UI so operators have a better idea of what each fleet is doing, can do, or will do.

I don't know exactly what method Morgan has in mind for gathering those navigation graphs, but I could imagine that we may have the fleet adapters report their navigation graphs to the map server when they start up.